src/devices/machine/mb8421.h

```diff
@@ -85,7 +85,7 @@ class mb8421_master_device : public device_t
 	DECLARE_READ_LINE_MEMBER(busy_r) { return 0; } // _BUSY pin - not emulated
 
 protected:
-	mb8421_master_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock);
+	mb8421_master_device(const machine_config &mconfig, device_type type, const char *tag, device_t *owner, u32 clock, size_t ram_size, u64 data_mask = ~0);
```

Comment:
The name isn't really appropriate when it's being used for a family of devices.  It should probably be called `dual_port_mailbox_ram_base` or something.